frontends/web/src/routes/device/bitbox01/setup/seed-create-new.jsx

```diff
@@ -48,13 +48,16 @@ class SeedCreateNew extends Component {
         },
     }
 
+    walletNameInput = createRef();
+    backupPasswordInput = createRef();
+
     componentDidMount () {
         this.checkSDcard();
     }
 
     validate = () => {
         // @ts-ignore
-        if (!this.walletNameInput || !this.walletNameInput.validity.valid || !this.validAgreements()) {
+        if (!this.walletNameInput.current || !this.walletNameInput.current.validity.valid || !this.validAgreements()) {
```

Comment:
I get an error here, not sure why this happens 🤔 

![Screen Shot 2021-12-01 at 6 50 29 PM](https://user-images.githubusercontent.com/546900/144287116-1427c797-e3b1-41ba-b922-242e5d83c928.png)

Reply:
This was because i was not forwarding the ref, fixed when changed to refInput prop

Reply:
It works now for transaction note `<Input>` 👍 , but here and a few other places in the BB01 setup it is using a different component `<PasswordInput>` and `<PasswordRepeatInput>`. Both use Input internally, and for focus these 2 components could forward the inputRef. But the ref is also used to access some internal function of the component 🙈 

I believe there 3 have still the issue

![Screen Shot 2021-12-02 at 10 55 00 PM](https://user-images.githubusercontent.com/546900/144509637-a7cf5341-b77a-4161-9548-d44d50d707c6.png)